sched/Kconfig

```
@@ -595,6 +595,12 @@ config SCHED_WAITPID
 		compliant) and will enable the waitid() and wait() interfaces as
 		well.
 
+config SCHED_DUMP_LEAK
+	bool "Enable catch task memory leak"
+	default n
+	---help---
+		Catch memory leaks when the task exits.
```

Improve the description? Suggestion:

```suggestion
		When this option is enabled, the task's outstanding memory allocations
		are printed using syslog. This helps catch any memory allocated by the
		task that remains unreleased when the task exits.
```

---

Done~